source/create_synonyms_and_grants_for_public.sql

```diff
@@ -88,6 +88,7 @@ create public synonym ut_runner for &&ut3_owner..ut_runner;
 create public synonym ut_teamcity_reporter for &&ut3_owner..ut_teamcity_reporter;
 create public synonym ut_xunit_reporter for &&ut3_owner..ut_xunit_reporter;
 create public synonym ut_documentation_reporter for &&ut3_owner..ut_documentation_reporter;
+create public synonym ut_ansiconsole_helper for &&ut3_owner..ut_ansiconsole_helper;
```

Comment:
I would remove this synonym. ut_ansiconsole_helper is `internal` code and we don't need to make it publicly available.
Not exposing it publicly gives us more freedom to change internals of the project.